modules/cold-extraction/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.1'
+services:
+  niffler-cold-extraction:
+    build: .
+    volumes:
+    - ./config.json:/usr/src/niffler-cold-extraction/config.json
+    - ./system.json:/usr/src/niffler-cold-extraction/system.json
+    - ./csv/:/usr/src/niffler-cold-extraction/csv/
+    - ./storage:/opt/data/new-study
+    networks:
+      - default
+  orthancc:
```

Comment:
We don't really use Orthanc in cold-extraction module. Orthanc should be removed from here.

Reply:
Noted, I can remove it. It was mentioned in the README and it's useful for testing so I thought to include it.

Reply:
That is a fair point regarding README. Just it is not really part of the cold-extraction module itself as of now. You can remove it for the sake of consistency.